tests/AnonymousUserTests/MoqAutoDataAttribute.cs

```diff
@@ -0,0 +1,43 @@
+using System.Security.Claims;
+using AutoFixture;
+using AutoFixture.AutoMoq;
+using AutoFixture.NUnit3;
+using InsightArchitectures.AnonymousUser;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Internal;
+using Moq;
+
+namespace AnonymousUserTests
+{
+    public class MoqAutoDataAttribute : AutoDataAttribute
+    {
+        public MoqAutoDataAttribute() : base(() =>
```

Comment:
I usually follow this template. It makes it easier to quickly introduce InlineAutoData attributes if needed.

https://github.com/insight-architectures/templates/blob/master/content/CustomAutoDataAttributes/CustomAutoDataAttributes.cs

Also, FWI, you can use this template package https://www.nuget.org/packages/InsightArchitectures.Templates/ (source [here](https://github.com/insight-architectures/templates)) to quickly introduce reusable parts :)